src/interview/oracle/SwapNodes.java

```diff
@@ -0,0 +1,117 @@
+package interview.oracle;
+
+import java.io.*;
+import java.util.*;
+import java.text.*;
+import java.math.*;

```

Comment:
Written by Amit and would be quite confusing.Should write code which can be easily understandable.